scss/_base_forms-tick-elements.scss

```diff
@@ -18,40 +23,114 @@ $box-offsets-top: (
   table-header: 1.9,
 ) !default;
 
-@mixin vf-b-tick-elements {
-  // Default form checkbox and radio styles
-  %vf-tick-elements {
+@mixin vf-b-tick-elements-definitions {
+  %vf-hidden-tick-input {
+    bottom: 0;
+    float: none;
+    height: $form-tick-box-size;
+    margin: 0;
     opacity: 0;
     position: absolute;
+    width: $form-tick-box-size;
+  }
 
-    label & {
-      // some basic styling for checkboxes inside labels
-      float: left;
-      margin: $spv-inner--x-small $sph-inner 0 0;
+  %vf-pseudo-tick-box {
+    padding-left: $sph-inner + $form-tick-box-size;
+    position: relative;
+
+    &::before,
+    &::after {
+      $properties: #{background-color, border-color};
+      @include vf-animation($properties);
+
+      position: absolute;
+    }
+
+    // container
+    &::before {
+      content: '';
+      height: $form-tick-box-size;
+      left: 0;
+      outline-offset: 1px;
+      width: $form-tick-box-size;
+    }
+
+    // tick/circle
+    &::after {
+      content: '';
+      opacity: 0;
+    }
+  }
+
+  %vf-pseudo-tick-box-checked {
+    &::before {
+      background-color: $color-link;
+      border-color: $color-link;
+    }
+
+    &::after {
       opacity: 1;
-      position: static;
     }
+  }
 
-    & + label {
-      padding-left: $sph-inner + $box-size;
-      position: relative;
+  %vf-pseudo-tick-box-focused {
+    &::before {
+      outline: $bar-thickness solid $color-focus;
+    }
+  }
 
-      &::before,
-      &::after {
-        $properties: #{background-color, border-color};
-        @include vf-animation($properties);
+  %vf-pseudo-checkbox {
+    // container
+    &::before {
+      border-radius: $border-radius;
+    }
 
-        position: absolute;
-      }
+    //tick
+    &::after {
+      border-bottom: 2px solid;
+      border-left: 2px solid;
+      height: $form-tick-height;
+      left: $form-tick-height * 0.5;
+      transform: rotate(-45deg);
+      width: 0.625rem;
+    }
+  }
 
-      // container
-      &::before {
-        content: '';
-        height: $box-size;
-        left: 0;
-        outline-offset: 1px;
-        width: $box-size;
-      }
+  %vf-pseudo-radio {
+    &::before,
+    &::after {
```

Comment:
border-radius is defined on 106 too - should one be dropped? I think separate makes it easier to work with

Reply:
Oh, didn't notice that. I just copied what we had before.